deployments/liqo/values.yaml

```diff
@@ -169,7 +169,7 @@ webhook:
     annotations: {}
     # -- The label that needs to be applied to a namespace to make it eligible for pod offloading in a remote cluster
     namespaceSelector:
-      liqo.io/enabled: "true"
+      liqo.io/scheduling: "true"
```

Comment:
I have found an occurrence of it in `pkg/consts/namespace_mapping.go` can you please check if we can update it and if it is used?

And I think that we will update it in the documentation when releasing the 0.3 version @palexster